domains/samarth.json

```diff
@@ -1,9 +1,9 @@
 {
   "owner": {
-    "username": "Samarth2005",
-    "email": "samarthkulshresth@gmail.com"
+    "username": "samarthkulshrestha",
```

Comment:
Please get @Samarth2005 to confirm this change.

Reply:
`Samarth2005` was my old GitHub username that I later changed to `samarthkulshrestha`. I think it has now been reassigned to someone else. I don't really know how to confirm this. Could you suggest way? Thank you.

Reply:
@samarthkulshrestha Nevermind, I checked the file history and your new name is there.